spec/classes/plugin/dns_rfc2136_spec.rb

```diff
@@ -21,6 +21,8 @@ class { 'letsencrypt':
           'python3-certbot-dns-rfc2136'
         when 'RedHat-7', 'CentOS-7'
           'python2-certbot-dns-rfc2136'
+        when 'FreeBSD-12', 'FreeBSD-13'
+          'py38-certbot-dns-rfc2136'
```

Comment:
On line 48 bellow, config file path is hardcoded as `/etc/letsencrypt/dns-rfc2136.ini`, I would expect something under `/usr/local/etc` on FreeBSD :wink:

Reply:
https://github.com/voxpupuli/puppet-letsencrypt/pull/268/commits/1448922831d3516ba1084cc8fae891d0a7bd4358 should do the trick.